integration-test.Rmd

```diff
@@ -37,6 +37,13 @@ suppressPackageStartupMessages(library(fs))
 library(here)
 ```
 
+```{r}
+library(conflicted)
+conflicted::conflict_prefer("filter", "dplyr")
+conflicted::conflict_prefer("flatten_chr", "purrr")
+conflicted::conflict_prefer("lag", "dplyr")
```

Comment:
I don't know if this will solve any of the errors we see now, and I don't know if the way I defined the preference is the intended by the author of the code that uses those functions. But this is a first step towards being explicit about potential conflicts.

Reply:
To keep PRs small and focused, I move this conversation to https://github.com/2DegreesInvesting/PACTA_analysis/pull/128.